suite-native/atoms/src/AlertBox.tsx

```diff
@@ -1,46 +1,101 @@
+};
+
+const alertWrapperStyle = prepareNativeStyle<AlertWrapperStyleType>(
+    (utils, { isStandalone, backgroundColor }) => ({
+        flexDirection: 'row',
+        alignItems: 'center',
+        borderRadius: isStandalone ? utils.borders.radii.small : 0,
+        backgroundColor: utils.colors[backgroundColor],
+        paddingVertical: utils.spacings.small,
+        paddingHorizontal: utils.spacings.medium,
+        gap: utils.spacings.small,
+    }),
+);
 
-const textWidthStyle = prepareNativeStyle(_ => ({
+const textStyle = prepareNativeStyle(_ => ({
     flex: 1,
 }));
 
-export const AlertBox = ({ title, isIconVisible = true }: AlertBoxProps) => {
+const variantToColorMap = {
+    info: {
+        backgroundColor: 'backgroundAlertBlueSubtleOnElevation0',
+        contentColor: 'iconAlertBlue',
+    },
+    success: {
+        backgroundColor: 'backgroundPrimarySubtleOnElevation0',
+        contentColor: 'textSecondaryHighlight',
+    },
+    warning: {
+        backgroundColor: 'backgroundAlertYellowSubtleOnElevation0',
+        contentColor: 'textAlertYellow',
+    },
+    error: {
+        backgroundColor: 'backgroundAlertRedSubtleOnElevation0',
+        contentColor: 'textAlertRed',
+    },
+} as const satisfies Record<AlertBoxVariant, AlertBoxStyle>;
+
+const variantToIconName = {
+    info: 'info',
+    success: 'checkCircle',
+    warning: 'warningTriangle',
+    error: 'warningTriangle',
+} as const satisfies Record<AlertBoxVariant, IconName>;
+
+export const AlertBox = ({
+    title,
+    isStandalone = false,
+    variant = 'info',
+    textAppendix,
+}: AlertBoxProps) => {
     const { applyStyle } = useNativeStyles();
 
     return (
-        <Box style={applyStyle(alertWrapperStyle)}>
-            {isIconVisible && (
-                <Box marginRight="medium">
-                    <RoundedIcon
-                        name="info"
-                        color="iconAlertBlue"
-                        iconSize="medium"
-                        backgroundColor="backgroundAlertBlueSubtleOnElevation1"
-                    />
-                </Box>
-            )}
-            <Box style={applyStyle(textWidthStyle)}>
-                <Text color="textAlertBlue" textAlign={isIconVisible ? 'left' : 'center'}>
-                    {title}
-                </Text>
-            </Box>
+        <Box
+            style={applyStyle(alertWrapperStyle, {
+                isStandalone,
+                backgroundColor: variantToColorMap[variant].backgroundColor,
+            })}
+        >
+            <Icon
+                name={variantToIconName[variant]}
+                color={variantToColorMap[variant].contentColor}
+                size="mediumLarge"
+            />
+            <Text
+                color={variantToColorMap[variant].contentColor}
+                variant="label"
+                style={applyStyle(textStyle)}
+            >
+                {title}
+                {textAppendix && ' '} {textAppendix}
```

Comment:
What's are benefits of this? I think using just `title` should be enough

Reply:
using Reactnode for title should probably solve this usecase. good point

Reply:
e5deb4e